src/main/java/com/cleanroommc/groovyscript/compat/mods/aether_legacy/Accessory.java

```diff
@@ -0,0 +1,112 @@
+package com.cleanroommc.groovyscript.compat.mods.aether_legacy;
+
+import com.cleanroommc.groovyscript.api.GroovyLog;
+import com.cleanroommc.groovyscript.api.documentation.annotations.*;
+import com.cleanroommc.groovyscript.helper.Alias;
+import com.cleanroommc.groovyscript.helper.SimpleObjectStream;
+import com.cleanroommc.groovyscript.helper.recipe.AbstractRecipeBuilder;
+import com.cleanroommc.groovyscript.registry.ForgeRegistryWrapper;
+import com.cleanroommc.groovyscript.registry.ReloadableRegistryManager;
+import com.gildedgames.the_aether.api.accessories.AccessoryType;
+import com.gildedgames.the_aether.api.accessories.AetherAccessory;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+import org.jetbrains.annotations.Nullable;
+
+@RegistryDescription
+public class Accessory extends ForgeRegistryWrapper<AetherAccessory> {
+
+    private static final AccessoryType[] VALUES = AccessoryType.values();
+
+    public Accessory() {
+        super(GameRegistry.findRegistry(AetherAccessory.class), Alias.generateOf("accessory"));
```

Comment:
this should be capitalized to properly generate the options, or you can set it to generate based on the class name, which will do so automatically. this will likely also require renaming the lang keys from eg `freezerfuel` -> `freezer_fuel`
```suggestion
        super(GameRegistry.findRegistry(AetherAccessory.class), Alias.generateOfClass(Accessory.class));
```